client/src/main/java/org/spine3/validate/Validate.java

```diff
@@ -107,10 +111,57 @@ public static boolean isNotDefault(Message object) {
      * @param object the {@code Message} instance to check
      * @throws IllegalStateException if the object is in its default state
      */
+    public static <M extends Message> M checkNotDefaultState(M object) {
+        checkNotNull(object);
+        checkNotDefaultState(object,
+                             "The message is in the default state: %s",
+                             TypeName.of(object));
+        return object;
+    }
+
+    /**
+     * Ensures that the passed object is not default and is not {@code null}.
+     *
+     * @param object    the {@code Message} instance to check
+     * @param errorMsg  the message for the exception to be thrown;
+     *                  will be converted to a string using {@link String#valueOf(Object)}
+     * @throws IllegalArgumentException if the object is in its default state
+     */
+    public static <M extends Message> M checkNotDefault(M object, @Nullable Object errorMsg) {
+        checkNotNull(object);
+        checkArgument(isNotDefault(object), errorMsg);
+        return object;
+    }
+
+
+
+    /**
+     * Ensures that the passed object is not default and is not {@code null}.
+     *
+     * @param object               the {@code Message} instance to check
+     * @param errorMessageTemplate a template for the exception message should the check fail
+     * @param errorMessageArgs     the arguments to be substituted into the message template
+     * @throws IllegalArgumentException if the object is in its default state
+     */
+    @SuppressWarnings("OverloadedVarargsMethod")
+    public static <M extends Message> M checkNotDefault(M object, String errorMessageTemplate,
+                                                        Object... errorMessageArgs) {
```

Comment:
Please do not ignore the warning but deal with it instead.
It's not quite obvious which exact method is called in this case.